zeth.txt

```diff
@@ -0,0 +1,14 @@
+[Section]
+Name = Zeth
+Version = 0.0.1
+License = Freeware
+Description = Zeth Visual Style for ReactOS
+Size = 116 KiB
+Category = 15
+URLDownload = https://github.com/Mas4hmad/ms-visual-styles/raw/master/ZETH.msstyles
+SHA1 = D06F5C1ADB15FB892F07583133D02E23AC3575C7
```

Comment:
Are you sure this is the correct hash?

Reply:
oh, nope. i input that SHA1 before commit into repo. i don't know if it SHA1 is different. will do correct